app/views/layouts/listnav/_container_project.html.haml

```diff
@@ -71,4 +70,14 @@
             %li
               = link_to(_("Replicators (%s)") % num_replicators,
                   {:action  => 'show', :id => @record, :display => 'container_replicators'},
-                  :title => _("Show Replicators"))
\ No newline at end of file
+                  :title => _("Show Replicators"))
+        - if role_allows(:feature => "container_node_show")
+          - num_nodes = @record.number_of(:container_nodes)
+          - if num_nodes == 0
+            %li.disabled
+              = link_to(_("Nodes (0)"), "#")
+          - else
+            %li
+              = link_to(_("Nodes (#{num_nodes})"),
+                {:action => 'show', :id => @record, :display => 'container_nodes'},
+                 :title  => _("View %s") % ui_lookup(:table => "container_nodes"))
```

Comment:
@moolitayer should be changed to ui_lookup(:tables => "container_nodes") can we use ui_lookup in other places as well to display title and hover text.